labs/provision-federation-controller-manager.md

```diff
@@ -57,8 +57,11 @@ kubectl config set-context federation-cluster \
 Switch to the `federation-cluster` context and dump the federated API server credentials:
 
 ```
-kubectl --context=federation-cluster \
-  config view --flatten --minify > kubeconfigs/federation-apiserver/kubeconfig
+kubectl config use-context federation-apiserver
+```
```

Comment:
I believe this should be "kubectl config use-context federation-cluster" and not "kubectl config use-context federation-apiserver". I get an error with the former "no context exists with the name: "federation-apiserver"."